src/main/java/com/worldwarofants/game/module/startmenu/StartMenuController.java

```diff
@@ -2,9 +2,11 @@
 
 import com.worldwarofants.game.arch.AbstractController;
 import com.worldwarofants.game.arch.module.IModuleNavigator;
+import com.worldwarofants.game.model.World;
 import com.worldwarofants.game.module.ModuleName;
 
 public class StartMenuController extends AbstractController<StartMenuView> {
+	private World w = new World();
```

Comment:
A world object is already created in ConsoleGame, you should use that one instead of creating a new one. 

And the cleaner way to access the world class is to use a service class. You can look at the example service in the code to read what it does.